File: src/nl/rubensten/texifyidea/action/insert/InsertSectionAction.java
package nl.rubensten.texifyidea.action.insert;

import nl.rubensten.texifyidea.TexifyIcons;
import nl.rubensten.texifyidea.action.InsertEditorAction;

/**
 * @author Ruben Schellekens
 */
public class InsertSectionAction extends InsertEditorAction {

    public InsertSectionAction() {
        super("Section", TexifyIcons.DOT_SECTION, "\\section{", "}");
    }
}

File: src/nl/hannahsten/texifyidea/action/insert/InsertSectionAction.kt
package nl.hannahsten.texifyidea.action.insert;

import nl.hannahsten.texifyidea.TexifyIcons;
import nl.hannahsten.texifyidea.action.InsertEditorAction;

/**
 * @author Hannah Schellekens
 */
public class InsertSectionAction extends InsertEditorAction {

    public InsertSectionAction() {
        super("Section", TexifyIcons.DOT_SECTION, "\\section{", "}");
    }
}
